firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/TesterSignInManager.java

```diff
@@ -111,6 +111,39 @@ void onActivityStarted(Activity activity) {
     }
   }
 
+  @NonNull
```

Comment:
Just a reminder: the changes to this file can be reverted

Reply:
Yup did a fast follow.